net-print/cnijfilter-drivers/files/cnijfilter-3.70-libexec-cups.patch

```diff
@@ -0,0 +1,9 @@
+diff -urN cnijfilter-source-3.70-1.old/pstocanonij/filter/Makefile.am cnijfilter-source-3.70-1/pstocanonij/filter/Makefile.am
```

Comment:
Combine the patches that do the same thing. Also, would be nice to have some explanation what a patch does on the top and why.

Reply:
Well... we could combine these patches, for sure, but I am currently in the phase of recovering as many proposed patches as possible from many guys in many locations, and I would like to keep the traceability by keeping the patch names and contents in that phase. Otherwise it would become practically impossible to consolidate that long history.

Reply:
Closed